pysat/instruments/sw_f107.py

```diff
@@ -381,7 +383,8 @@ def download(date_array, tag, sat_id, data_path, user=None, password=None):
                 data.iloc[idx, :] = np.nan
                 # create file
                 data.to_csv(os.path.join(data_path, 'f107_monthly_' +
-                                         date.strftime('%Y-%m') + '.txt'))
+                                         date.strftime('%Y-%m') + '.txt'),
+                            header=True)
```

Comment:
This will impact Angeline's current F107 data set. Better moving forward but it is technically backwards incompatible.

Reply:
I can pull out the `header=True` statements, but it looks like the `sw_f107` code in the develop branch is producing headers by default (at least on 3.6). @aburrell, can you confirm?

Reply:
All the files I know of have headers.